yt_dlp/options.py

```diff
@@ -1725,14 +1725,15 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         '--convert-subs', '--convert-sub', '--convert-subtitles',
         metavar='FORMAT', dest='convertsubtitles', default=None,
         help=(
-            'Convert the subtitles to another format (currently supported: {}) '
-            '(Alias: --convert-subtitles)'.format(', '.join(sorted(FFmpegSubtitlesConvertorPP.SUPPORTED_EXTS)))))
+            'Convert the subtitles to another format '
+            f'(currently supported: {", ".join(sorted(FFmpegSubtitlesConvertorPP.SUPPORTED_EXTS))}, none) '
+            '(Alias: --convert-subtitles)'))
     postproc.add_option(
         '--convert-thumbnails',
         metavar='FORMAT', dest='convertthumbnails', default=None,
         help=(
             'Convert the thumbnails to another format '
-            f'(currently supported: {", ".join(sorted(FFmpegThumbnailsConvertorPP.SUPPORTED_EXTS))}). '
+            f'(currently supported: {", ".join(sorted(FFmpegThumbnailsConvertorPP.SUPPORTED_EXTS))}, none). '
             'You can specify multiple rules using similar syntax as --remux-video'))
```

Comment:
tacking `none` on to the end of the extension list is misleading here IMO, since the following line then implies you can use it like `--convert-thumb "jpg>png/webp>none"`
```suggestion
            f'(currently supported: {", ".join(sorted(FFmpegThumbnailsConvertorPP.SUPPORTED_EXTS))}). '
            'You can specify multiple rules using similar syntax as "--remux-video". '
            'A previous pass of this option can be negated with "--convert-thumbnails none"'))
```

Reply:
Fair enough! Even though subtitles don't have the same caveat, would you like me to use the same language there for consistency?

Reply:
If you want to. I'm not exactly thrilled with "a previous pass of this option" as the wording, though

Reply:
It's hard to convey succinctly! Thoughts on `Use "--convert-thumbnails none" to disable conversion (default)`? This would rely on the user knowing that options will be overwritten if they're repeated

Reply:
```suggestion
            f'(currently supported: {", ".join(sorted(FFmpegThumbnailsConvertorPP.SUPPORTED_EXTS))}). '
            'You can specify multiple rules using similar syntax as "--remux-video". '
            'Use "--convert-thumbnails none" to disable conversion (default)'))
```